runtimes/common/src/constants.rs

```diff
@@ -218,27 +228,50 @@ pub mod did {
 	pub const MAX_NUMBER_OF_URLS_PER_SERVICE: u32 = 1;
 }
 
-pub mod treasury {
-	use crate::{Balance, BlockNumber};
+pub mod did_lookup {
+	use super::*;
 
-	use super::{BLOCKS_PER_YEAR, KILT};
+	pub const CONNECTION_SIZE: u32 = 80;
+	pub const DID_CONNECTION_DEPOSIT: Balance = deposit(1, CONNECTION_SIZE);
+}
+
+pub mod treasury {
+	use super::*;
 
 	pub const INITIAL_PERIOD_LENGTH: BlockNumber = BLOCKS_PER_YEAR.saturating_mul(5);
 	const YEARLY_REWARD: Balance = 2_000_000u128 * KILT;
 	pub const INITIAL_PERIOD_REWARD_PER_BLOCK: Balance = YEARLY_REWARD / (BLOCKS_PER_YEAR as Balance);
 }
 
 pub mod proxy {
-	pub const MAX_PROXIES: u16 = 10;
-	pub const MAX_PENDING: u16 = 10;
+	use super::*;
+
+	parameter_types! {
+		// One storage item; key size 32, value size 8; .
+		pub const ProxyDepositBase: Balance = deposit(1, 8);
+		// Additional storage item size of 33 bytes.
+		pub const ProxyDepositFactor: Balance = deposit(0, 33);
+		pub const MaxProxies: u16 = 10;
+		pub const AnnouncementDepositBase: Balance = deposit(1, 8);
+		pub const AnnouncementDepositFactor: Balance = deposit(0, 66);
+		pub const MaxPending: u16 = 10;
+	}
 }
 
 pub mod web3_names {
-	use crate::Balance;
-
-	use super::KILT;
+	use super::*;
 
 	pub const MIN_LENGTH: u32 = 3;
 	pub const MAX_LENGTH: u32 = 32;
-	pub const DEPOSIT: Balance = 2 * KILT;
+
+	pub const NAME_SIZE: u32 = 121;
```

Comment:
I would call it `NAME_MAX_SIZE`.

Reply:
I would not. I would need to add MAX everywhere.

Reply:
But that's what it is, isn't it?

Reply:
Should I also add bytes since it's bytes?

Reply:
[c5a8177](https://github.com/KILTprotocol/mashnet-node/pull/329/commits/c5a8177fc4e98b017c9c87028d7db028aaafc1bc)

Reply:
I changed `size` to `byte_length` since that terminology is also used somewhere else. [153e738](https://github.com/KILTprotocol/mashnet-node/pull/329/commits/153e7386734e4a61a3e830df20e3278510d7cc70)